ted_sws/event_manager/services/logger_from_context.py

```diff
@@ -47,6 +35,35 @@ def get_env_logger(logger: EventLogger, is_cli: bool = False) -> EventLogger:
         return EventLogger(NULLLoggerConfig())
```

Comment:
Why NULLLoggerConfig(), I recommend to rename it.

Reply:
why? null is emptiness/void/nothing. it's pretty obvious for me that this logger instance will send the logs to nothing/nowhere

Reply:
logging.NullHandler()
this is the native Python Handler to be used for testings purposes.
It's a very good name. I will rename it to NullLoggerConfig :D

Reply:
echo "" > /dev/null

Reply:
Null is better than NULL in class name 👍